### DISCUSSION TITLE
Newly entered habits showing six months of old, incompleted activity

### DISCUSSION THREAD

**Discussion by spiraldancing** [iSoron/uhabits#698] [Help & FAQ]
Trying to reset / restart for the new year ... there doesn't seem to be any way to add new habits and have the app just start tracking my history for the new habits, on the day I enter them.

So, I uninstalled the app, and then reinstalled it ... starting from scratch, right?  Nope.  Brand new set-up, add new habits, and the app shows I have "not done" my new habits for the past 5-6 months.

Am I missing something here?  Is this a bug or a feature?  Any way around this?

PS:  I've been using this app for a couple of years now ... it is great.  Thanks for the ongoing work.

**Comment by iSoron** [comment]
> the app shows I have "not done" my new habits for the past 5-6 months

Hi @spiraldancing, could you clarify what do you mean by this? What charts/screens make you think that the app started tracking your habit 5-6 months ago? Screenshots would be helpful. Currently, Loop does not have starting or ending dates for habits. They all extend infinitely into the past (and also into the future).

**Reply by amrsamii** [reply]
Will you support starting and ending dates for habits? I think it will be good as scores will be more realistic.

**Comment by spiraldancing** [comment]
>  They all extend infinitely into the past (and also into the future).

That is probably the answer I needed, but to clarify --

I uninstalled/reinstalled the app yesterday, 1/2/21.  I kind of expected/assumed that there would only be checkboxes available for that date moving forward, but what I see are checkboxes going all the way back to Nov 4th (oddly, I thought it went even further back, to August or Sept, when I started this thread yesterday).

I don't seem to be able to upload screenshots here (maybe GitHub, maybe one of my browser plug-ins, maybe I'm just ignorant) ... at any rate, [here is a link](https://99acrewood.xyz/index.php/s/Ac88oQyLP233fyG) to the screenshots on my NextCloud site.

I see something similar in the detail screen of each habit, showing that I have accomplished 0% of this habit going back several months (probably infinitely, as you noted).

ETA:  For future visitors, sorry, but I'll be killing that URL link in a week or so...
